### DISCUSSION TITLE
How do I disable fullReload?

### DISCUSSION THREAD

**Discussion by firedev** [ElMassimo/vite_ruby#383] [General]
Hi, is there a way to stop vite from reloading the page? The app is quote heavy and each time it reloads I have to wait. I can't seem to stop it from happening. I uninstalled the `vite-plugin-full-reload` package, it's happening anyway. Please advise.

**Comment by mtomov** [comment]
Hey @firedev 

Yes, I came across the same issue. Ssee here:

https://github.com/ElMassimo/vite-plugin-full-reload/issues/2
